src/components/NavbarStudyStudio.scss

```diff
@@ -128,3 +128,9 @@ $content-color: #1b1b1f;
 .svg__item--selected {
   fill: $white-color;
 }
+
+.all-right{
```

Comment:
Please, add spaces:
1. Before '{' in '.all-right{'

2. After ':' in 'bottom:0;'

Please, find possibility to avoid such in future/ You can try plugins in VS Code that auto fix this